feature-libs/asm/components/asm-main-ui/asm-main-ui.component.ts

```diff
@@ -132,6 +162,30 @@ export class AsmMainUiComponent implements OnInit, OnDestroy {
           }
         })
     );
+    this.subscribeForDeeplink();
+  }
+
+  private subscribeForDeeplink(): void {
+    if (this.router) {
```

Comment:
According to the discussion with Jason and @luoyifeng11, if I understand correctly, customer has a flag representing the version he/she want to use. That means a customer could be able to upgrade his/her code to new version and keep the flag as old version. It will help he/she to fix some bug that being fixed in new versions and not introduce any new features. In this case, just using `if (this.router)` is not enough. We had better use `featureConfig` instead of this.
@luoyifeng11 Please help confirm if this is the case.  Thx.

Reply:
I see in the PR description it says, 'Feature level will be added later to keep compatibility for old versions.' Does this mean that the featureLevel will be added later? @scarai-sap

Reply:
I just added when refactor the code.